tsrc/cli/foreach.py

```diff
@@ -90,19 +87,13 @@ def die(message: str) -> None:
     sys.exit(1)
 
 
-@with_workspace  # type: ignore
-@with_groups  # type: ignore
-@with_all_cloned  # type: ignore
-@arg("cmd", help="command to run", nargs="*")  # type: ignore
-@arg("-c", help="use a shell to run the command", dest="shell")  # type: ignore
-def foreach(
-    cmd: List[str],
-    workspace_path: Optional[Path] = None,
-    groups: Optional[List[str]] = None,
-    all_cloned: bool = False,
-    shell: bool = False,
-) -> None:
-    """ run the same command on several repositories """
+@repos_arg
+@repos_action
```

Comment:
Shouldn't you ignore typing from these too? Their signature is (Callable) -> Callable, it seems to me that they are more hurtful than helpful. `type: ignore` would keep the signature of the function as it is defined below, right?